include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -98,7 +98,7 @@ def InstanceOp : ReferableDeclOp<"instance", [HasParent<"firrtl::FModuleOp, firr
   }];
 }
 
-def MemOp : ReferableDeclOp<"mem"> {
+def MemOp : ReferableDeclOp<"mem", [FNamableOp]> {
```

Comment:
Should FNamableOp just be added to the base ReferableDeclOp? I see it isn't on InstanceOp now, but maybe it could be there too?

Reply:
Good point. I intentionally avoided InstanceOp because the dropname canonicalizer is not implemented for InstanceOp name. But on the second thought, there should be no reason to exclude InstanceOp so I'll move FNamebleOp into ReferrableOp.